cmd/skuba/main.go

```diff
@@ -79,6 +79,8 @@ func register(local *pflag.FlagSet, globalName string) {
 	if f := flag.CommandLine.Lookup(globalName); f != nil {
 		pflagFlag := pflag.PFlagFromGoFlag(f)
 		pflagFlag.Name = "verbosity"
+		pflagFlag.Usage = "number for the log level verbosity [0-10]"
+		local.PrintDefaults()
```

Comment:
What is the intent of this line?

Reply:
local.PrintDefaults() shows default value of verbosity level in specific cases when I tested it.
when I  used "skuba -v 1" or 'skuba -h v 2', it showed the default values. 
'skuba -h' does not show the default value.
If you think this is not useful, I can delete it.
Let me know.

Reply:
This call should not be here, because it's conflating the building of the arguments with printing help information about it. I don't see any difference on the output of `skuba -h` and `skuba -v1 -h` with and without this patch in this specific regard (showing the `default` value). On current master:

```
~/p/g/s/g/S/skuba (master) > skuba -h
(snip)
  -v, --verbosity Level   number for the log level verbosity
(snip)
```

```
~/p/g/s/g/S/skuba (master) > skuba -v1 -h
(snip)
  -v, --verbosity Level   number for the log level verbosity (default 0)
(snip)
```